test/module/irohad/torii/torii_service_test.cpp

```diff
@@ -209,10 +209,11 @@ TEST_F(ToriiServiceTest, StatusWhenBlocking) {
 
   // create transactions and send them to Torii
   std::string account_id = "some@account";
+  auto now = iroha::time::now();
   for (size_t i = 0; i < TimesToriiBlocking; ++i) {
     auto shm_tx = shared_model::proto::TransactionBuilder()
                       .creatorAccountId(account_id)
-                      .createdTime(iroha::time::now())
+                      .createdTime(now + i)
```

Comment:
I think this fix should be reworked with mock at the time.

Reply:
A task will be created.